### ISSUE TITLE
What part of the shim blocks svg:use + example

### ISSUE THREAD

**Issue by sheriffderek** [ember-cli/ember-cli-shims#68]
I was at a meetup last night ~ and mentioned my issues with SVG - _and waiting for Ember 2.7 for the removal of `baseURL`_ (which is supposedly blocking svg:use) – and someone said that when they remove **ember-cli-shims** that the SVGs show up again.

What part of the shims affect this?

To reproduce, one could 

1. drop this markup into a fresh application.hbs
2. serve + view
3. stop
4. remove ember-cli-shims
5. serve + view

```
{{!-- icon styles --}}
<style>
[class^="icon-"], [class*=" icon-"] {
    height: 32px;
    width: 32px;
    display: inline-block; 
    fill: currentColor;
    border: 1px solid red;
}
</style>

{{!-- the svg (hidden) --}}
<svg xmlns="http://www.w3.org/2000/svg" style="display: none;">
  
  <symbol id="icon-smile" viewBox="0 0 512 512">
    <path d="m361 307c-7 23-21 42-40 56c-19 14-41 21-65 21c-24 0-46-7-65-21c-19-14-33-33-40-56c-1-5-1-9 2-14c2-4 5-7 10-9c5-1 10-1 14 1c5 3 8 6 9 11c5 16 14 28 27 37c13 10 27 14 43 14c16 0 30-4 43-14c13-9 22-21 27-37c1-5 4-8 9-11c4-2 9-2 14-1c5 2 8 5 10 9c3 5 3 9 2 14z m-142-124c0 10-3 19-10 26c-7 7-16 10-26 10c-10 0-19-3-26-10c-7-7-11-16-11-26c0-10 4-19 11-26c7-7 16-11 26-11c10 0 19 4 26 11c7 7 10 16 10 26z m147 0c0 10-4 19-11 26c-7 7-16 10-26 10c-10 0-19-3-26-10c-7-7-10-16-10-26c0-10 3-19 10-26c7-7 16-11 26-11c10 0 19 4 26 11c7 7 11 16 11 26z m73 73c0-25-5-48-15-71c-9-23-22-42-39-58c-16-17-35-30-58-39c-23-10-46-15-71-15c-25 0-48 5-71 15c-23 9-42 22-58 39c-17 16-30 35-39 58c-10 23-15 46-15 71c0 25 5 48 15 71c9 23 22 42 39 58c16 17 35 30 58 39c23 10 46 15 71 15c25 0 48-5 71-15c23-9 42-22 58-39c17-16 30-35 39-58c10-23 15-46 15-71z m36 0c0 40-9 77-29 110c-20 34-46 60-80 80c-33 20-70 29-110 29c-40 0-77-9-110-29c-34-20-60-46-80-80c-20-33-29-70-29-110c0-40 9-77 29-110c20-34 46-60 80-80c33-20 70-29 110-29c40 0 77 9 110 29c34 20 60 46 80 80c20 33 29 70 29 110z"/>
  </symbol>
  
</svg>

{{!-- in use use --}}
<svg class="icon-x"><use xlink:href="#icon-smile"></use></svg>
```

**Comment by rwjblue** [comment]
This seems odd to me. Can you provide a demo repo?

**Comment by sheriffderek** [comment]
Last night, I did exactly as described above ~ and there was no SVG, and then there was after the removal --- I did the same thing this morning (trying to get a repo up for you) - and the SVG is there to start with. So mysterious... Sometimes I get the SVG on mobile Safari but not on desktop. 

I don't know... I'm just going to close this out until I can reproduce it / or get a public repo up with the behavior, But If it just works / that's even better.

Thanks.
